quinn/src/connection.rs

```diff
@@ -282,7 +282,12 @@ impl Connection {
     ///
     /// Streams are cheap and instantaneous to open unless blocked by flow control. As a
     /// consequence, the peer won't be notified that a stream has been opened until the stream is
-    /// actually used.
+    /// actually used. Calling [`open_bi()`] then waiting on the [`RecvStream`] without writing
+    /// anything to [`SendStream`] will never return.
```

Comment:
nit: such a future *will* return when the connection is closed.
```suggestion
    /// anything to [`SendStream`] will never succeed.
```
(and similarly below)